context-monitoring/src/main/java/de/atb/context/persistence/monitoring/MonitoringDataRepository.java

```diff
@@ -268,13 +254,10 @@ public synchronized Type getMonitoringData(final BusinessCase businessCase, fina
         return result;
     }
 
-    /*
+    /**
      * (non-Javadoc)
      *
-     * @see
-     * de.atb.proseco.persistence.IMonitoringDataRepository#getMonitoringData
-     * (de.atb.proseco.common.util.BusinessCase, java.lang.Class,
-     * java.lang.String)
+     * @see IMonitoringDataRepository#getMonitoringData(BusinessCase, java.lang.Class, java.lang.String)
```

Comment:
shouldn't there be : `* @see IMonitoringDataRepository#getMonitoringData(de.atb.context.common.util.ApplicationScenario, java.lang.Class, java.lang.String) `?

Reply:
changed